ivy/functional/backends/jax/elementwise.py

```diff
@@ -12,7 +12,13 @@
 from . import backend_version
 
 
-def abs(x: Union[float, JaxArray], /, *, out: Optional[JaxArray] = None) -> JaxArray:
+def abs(
+    x: Union[float, JaxArray],
+    /,
+    *,
+    out: Optional[JaxArray] = None,
+    where: Optional[ivy.Array] = None,
```

Comment:
Following the `numpy` convention, they've set the default value of `where` as `True`, hence it can't be typehinted as `Optional`. It should be `Union[bool, JaxArray]` instead of `Optional[ivy.Array]`